CHANGELOG.md

```diff
@@ -108,6 +108,7 @@
     Feature #4423: Rebalance soul gem values
     Task #4015: Use AppVeyor build artifact features to make continuous builds available
     Editor: New (and more complete) icon set
+    Bug #4424: [macOS] Cursor is either empty or garbage when compiled against macOS 10.13 SDK
```

Comment:
It was and it still is out of order, it'd look nicer if you put it with other bug-type issues before the two bugs that have higher number.

Reply:
@Capostrophic my bad, now I’m a bit more familiar with how we do changelogs. Thanks!

@psi29a thanks for fixing!